controllers/app.controller.js

```diff
@@ -55,3 +56,18 @@ exports.getCommentsByID = (req, res, next) => {
     res.status(200).send(result);
   });
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  if (!username || !body) {
+    return res.status(400).send({ msg: "Username and body are required" });
+  }
+  if (typeof username !== "string" || typeof body !== "string") {
+    return res.status(400).send({ msg: "Please Enter Valid Strings" });
+  }
```

Comment:
Both of the above if statements aren't needed as sql will throw an error. However, if you do want the custom error messages, both of these can be in the model with a Promise reject